packages/client/src/client/notifications.openfin.ts

```diff
@@ -335,15 +388,49 @@ export const registerCreditAcceptedNotifications = (
       handler || handleHighlightCreditBlotterAction,
     )
 
-    acceptedRfqWithQuote$.subscribe((rfq) => {
-      sendQuoteAcceptedNotification(rfq)
-    })
+    acceptedRfqWithQuoteSubscription = acceptedRfqWithQuote$.subscribe(
+      (rfq) => {
+        sendQuoteAcceptedNotification(rfq)
+      },
+    )
+  }
+}
+
+let areCreatedCreditNotificationsRegistered = false
+let createdCreditSubscription: Subscription | null = null
+
+const registerCreatedCreditNotification = () => {
+  if (!areCreatedCreditNotificationsRegistered) {
+    areCreatedCreditNotificationsRegistered = true
+    createdCreditSubscription = createdCreditConfirmation$.subscribe(
+      (rfqRquest) => {
+        sendRFQCreatedConfirmationNotification(rfqRquest)
+      },
+    )
   }
 }
 
-export const unregisterCreditQuoteNotifications = () => {
+export const registerCreditNotifications = (
```

Comment:
_(previously posted in the wrong place 🤦‍♂️)_
Here is where we will need to check `fin.Application.getCurrentSync().getXxx`
or whatever it is and do the logic to see if Launcher or Workspace is running when RT Credit starts up
..
Will need to do for FX as well, with moving to `LiveRatesCore`

Reply:
not so .. all notifications apart from Credit Quote derive from the ACKs to Hydra API calls, so will only occur once in the context that the call was initiated .. so as it stands we will just have the dup quote notifications as before